### PR TITLE
[Py] example missed import clauses

### PR DESCRIPTION
**Pull request by unam3**
### Description

Python code example missed import clauses.

### Motivation and Context

As a new user of the WebDriver I've spend almost an hour trying to figure out which modules should I import to run example. No links to documentation provided in the original pages which is unthinkable for project this big.

### Types of changes

- [ ] Change to the site (I have double-checked the Netlify deployment, and my changes look good)
- [x] Code example added (and I also added the example to all translated languages)
- [ ] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [x] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [x] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

### CODE REVIEW

File: website_and_docs/content/documentation/webdriver/getting_started/install_drivers.ja.md [hunk 1]
```diff
@@ -186,7 +186,13 @@ ChromeDriver driver = new ChromeDriver();
 
 {{< tab header="Python" >}}
 
+from selenium.webdriver.chrome.service import Service
+
 service = Service(executable_path="/opt/WebDriver/bin/chromedriver")
+
+
```
Comment (harsha509) [inline website_and_docs/content/documentation/webdriver/getting_started/install_drivers.ja.md]: Can we remove extra spaces in here.
one space between imports and code would be better

```
from selenium.webdriver.chrome.service import Service
from selenium import webdriver

service = Service(executable_path="/path/to/chromedriver")
driver = webdriver.Chrome(service=service)
```

**Review by harsha509 — CHANGES_REQUESTED** [review]

File: website_and_docs/content/documentation/webdriver/getting_started/install_drivers.ja.md (hunk 1 above)
Comment (unam3) [reply website_and_docs/content/documentation/webdriver/getting_started/install_drivers.ja.md]: doen

**Review by harsha509 — APPROVED** [review]
Thank you @unam3 !

**Comment by harsha509** [comment]
Congratulations on your first contribution to Selenium docs @unam3 🎉 🎉 🎉 !

**Pull request merged**